src/catala/surface/parser.mly

```diff
@@ -14,15 +14,15 @@
   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
```

Comment:
I'll just assume everything here was just reformated and not modified more than adding the precedence to `law_article`